docs/cover.md

```diff
@@ -24,13 +24,13 @@ To install the MapboxNavigation framework in an application using [Swift Package
 
 1. Enter `https://github.com/mapbox/mapbox-navigation-ios.git` as the package repository and click Next.
 
-1. Set Rules to Version, Up to Next Major, and enter `${MINOR_VERSION}` as the minimum version requirement. Click Next.
+1. Set Rules to Version, Up to Next Major, and enter `${SHORT_VERSION}` as the minimum version requirement. Click Next.
```

Comment:
document.sh defines `SHORT_VERSION` but doesn’t replace it in the readme:

https://github.com/mapbox/mapbox-navigation-ios/blob/d331990147231d8a57bf953390d63b6b178baf9a/scripts/document.sh#L42